xsd/netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd

```diff
@@ -483,7 +483,7 @@ Rail transport, Roads and Road transport
 			</xsd:extension>
 		</xsd:complexContent>
 	</xsd:complexType>
-	<xsd:element name="VehicleEntrance" abstract="true" substitutionGroup="Entrance">
+	<xsd:element name="VehicleEntrance" abstract="true" substitutionGroup="SiteComponent">
```

Comment:
This kills the method resolution order (MRO) of the schema. Hence VehicleEntrance will be at the same level as Entrance, while what we want is having the inheritance.

Reply:
I think we have a problem already, because Entrance comes from SITE and VehicleEntrance probably should not be a SITE. Or do I read this wrong?

Reply:
It also might be one of the cases where we need an Entrance_ . Because having to restrictions in a row is not allowed in the inheritance. Should I try to model an Entrance_? According to one of the examples?

Reply:
Or we should do the inheritance through the VersionStructure elements. I think that only in the "leaves" the id stuff can be done once.

Reply:
I think this is a case where Entrance_ would make sense. In this situation I also want to figure out why (or not) Entrance_ may (or may not) be abstract.